### PR TITLE
Add blackhulk, upload image and install mac to README.md

### PR DESCRIPTION
**Pull request by duykhoa**
With :heart:  and :beers:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ https://github.com/lunixbochs/usercorn)), optimization engine.
 * A simple, elegant PHP mySQL database library
 * Language: PHP
 
+##### [Blackhulk](https://github.com/FancyBlog/blackhulk)
+* An awesome blog engine. Blog posts will be saved in a GitHub Issues.
+
+##### [Upload Image](https://github.com/duykhoa/upload_image)
```
Comment (nvquanghuy) [inline README.md]: Possible to put the library's name as title instead? @duykhoa 

```
[upload_image](https://github.com/duykhoa/upload_image)
```

File: README.md [hunk 2]
```diff
@@ -14,6 +14,18 @@ https://github.com/lunixbochs/usercorn)), optimization engine.
...
+* A tiny gem to upload image to upload.im
+
+##### [Install mac](https://github.com/duykhoa/install_my_mac)
```
Comment (nvquanghuy) [inline README.md]: Same thing for this one and below. `install_my_mac`, `sg_postcode`

**Comment by nvquanghuy** [comment]
Thanks @duykhoa !

:heart: :heart: :heart: :heart:

File: README.md (hunk 2 above)
Comment (duykhoa) [inline README.md]: okay, do it now :+1:

**Pull request merged**